hockeysdk/src/main/java/net/hockeyapp/android/objects/CrashDetails.java

```diff
@@ -56,6 +56,10 @@
 
     public CrashDetails(String crashIdentifier) {
         this.crashIdentifier = crashIdentifier;
+        appStartDate = new Date();
```

Comment:
These two lines are dangerous, since the values might not be true in this case, so they are not sensible default values. They should be `null`. The code causing the exception you're experiencing should be handling the `null` case instead to either gracefully handle it (not recommended since we rely on these fields) or throw a more reasonable exception.
Overriding the `CrashManager` / `ExceptionHandler` or building your own should produce the same valid crash report files.